src/System.Management.Automation/engine/remoting/client/remotingprotocolimplementation.cs

```diff
@@ -752,7 +752,7 @@ public void Dispose()
         /// Release all resources.
         /// </summary>
         /// <param name="disposing">If true, release all managed resources.</param>
-        protected void Dispose(bool disposing)
```

Comment:
This is an internal class and should be sealed.

Reply:
@PaulHigin PR to seal the class: #21218.

Reply:
> This is an internal class and should be sealed.

Completed in https://github.com/PowerShell/PowerShell/pull/24705.